### DISCUSSION TITLE
How to stop Telebot read old messages?

### DISCUSSION THREAD

**Discussion by ClassfiedFliqqzy** [eternnoir/pyTelegramBotAPI#1721] [Q&A]
I'm using telebot in python to start a conversation with the user and then ask some questions, the problem is that after running the script again, the bot listens for messages sent before the script was executed. Is there any way to make the bot filter only for messages sent in the last 10 minutes or something?

**Comment by Badiboy** [comment]
![image](https://user-images.githubusercontent.com/5613421/190192784-b160968b-8f1c-4716-a505-60785e5413cc.png)

In Telebot(...).

**Comment by coder2020official** [comment]
Using skip pending will skip all updates except the last one;
If you want to skip ALL updates - use [delete_webhook](https://pytba.readthedocs.io/en/latest/sync_version/index.html#telebot.TeleBot.delete_webhook) by passing drop_pending_updates=True
